lib/ansiblelint/rules/TaskManyArgs.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2016, Tsukinowa Inc. <info@tsukinowa.jp>
+# Copyright (c) 2018, Ansible Project
+
+from ansiblelint import AnsibleLintRule
+
+
+class TaskManyArgs(AnsibleLintRule):
+    id = '206'
+    shortdesc = 'Use ":" YAML format when arguments are over 4'
```

Comment:
I'd favour using YAML format in all cases.

ansible-review has a better solution to this, but relies on tests run before canonicalization.

Reply:
Given that changing this from 4 to 0 would have false positives with any `=` in a text string, and the ansible-review solution does exist, I'm leaning towards removing this rule.

Reply:
Removed TaskManyArgs for now